app/react/MetadataExtraction/TrainingHealthDashboard.tsx

```diff
@@ -0,0 +1,50 @@
+import { Translate } from 'app/I18N';
+import React from 'react';
+import { SuggestionsStats } from 'shared/types/suggestionStats';
+import { GridChart } from './GridChart';
+import { TrainingHealthLegend } from './TrainingHealthLegend';
+
+interface TrainingHealthDashboardProps {
+  stats?: SuggestionsStats;
+}
+
+const mapStats = (data: SuggestionsStats['data']) => [
+  {
+    color: '#5073CF',
+    value: data.labeledMatching,
+    label: { text: <Translate>Training</Translate> },
+    overlaying: true,
+  },
+  {
+    color: '#CD4C4C',
+    value: data.labeled,
```

Comment:
While trying it out, this category doesn't show up for me in the legend (but I see the color on visualization). Missing label?